src/pages/understand-stacks/stacking-using-CLI.md

```diff
@@ -65,7 +65,13 @@ curl -X POST https://stacks-node-api.xenon.blockstack.org/extended/v1/faucets/st
 
 ## Check Balance
 
-Confirm that the faucet transaction has completed by checking the balance of your address. The `-t` flag is used to indicate testnet. See the [CLI reference](/references/stacks-cli) for usage of flags.
+It will take some time for the transaction to confirm on the network.  You can check the progress at `explorer.stacks.co`
+
+```bash
+https://explorer.stacks.co/txid/<txId>?chain=testnet
+```
+
+Once you've had a nice cup of tea, confirm that the faucet transaction has completed by checking the balance of your address. The `-t` flag is used to indicate testnet. See the [CLI reference](/references/stacks-cli) for usage of flags.
```

Comment:
I appreciate the candid language, but because a transaction might be confirmed somewhat quickly (or shorter than you can make a cup of tea), I would prefer:

"Once your transaction has been confirmed, you can verify your balance. The `-t` flag is used ....`

Reply:
@jamie-mint would appreciate and update and a signature on the CLA 🙏🏼